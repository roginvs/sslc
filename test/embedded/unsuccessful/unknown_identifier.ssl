procedure start begin
    whatisthis;
end

// After the first error the compiler stops so this is ignored until the first error is fixed
procedure someproc2 begin 
    whatisthis2;
end