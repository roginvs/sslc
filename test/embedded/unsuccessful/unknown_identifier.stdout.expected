Set include directory: ../include
Compiling unknown_identifier.ssl
[Warning] <unknown_identifier.ssl>:8: End of input with no newline, supplemented newline
    end
[Error] <Semantic> <unknown_identifier.ssl>:2:16: Unknown name identifier whatisthis.

*** THERE WERE ERRORS (1 of them) ***
